ui/app/components/transit-form-show.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
```

Comment:
does this need to be imported or does the PR you have ready to go account for this new file?

Reply:
Ah! GOOD CATCH. Thank you. I need to amend.